Sources/SwiftDocC/Semantics/Metadata/Metadata.swift

```diff
@@ -61,6 +65,7 @@ public final class Metadata: Semantic, AutomaticDirectiveConvertible {
         "customMetadata"        : \Metadata._customMetadata,
         "callToAction"          : \Metadata._callToAction,
         "availability"          : \Metadata._availability,
+        "pageKind"              : \Metadata._pageKind,
```

Comment:
We'll need to update the below `validate` method to make

```
@Metadata {
   @PageKind
}
```

valid. We should probably also add a test case for that.